CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- Updated Bitmovin Player to `3.69.0`
```

Comment:
I think this update implicitly also raises the minimum required compile SDK version, let's add a changelog for this as well, maybe
```
- Increased minimum required `compileSdk` to `34`
```

Reply:
This is already there on line 14 🙂

Reply:
https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/59/commits/76d337d24598a84a4ad48cf63c5a8784e1c10c7d